pkg/query-service/model/v3/v3.go

```diff
@@ -786,6 +806,7 @@ func (m *MetricValueFilter) Clone() *MetricValueFilter {
 type BuilderQuery struct {
 	QueryName            string            `json:"queryName"`
 	StepInterval         int64             `json:"stepInterval"`
+	SpanSearchScope      SpanSearchScope   `json:"spanSearchScope,omitempty"`
```

Comment:
can there be multiple scopes that might come in future ?

Reply:
I have changed the functionality entirely, but it could have multiple scopes.